scripts/build_distro.sh

```diff
@@ -18,6 +18,22 @@ distro=$3
     cd ${topdir}/build/
 
     ROOTFS_DIR=${topdir}/build/${build}/tisdk-${distro}-${machine}-rootfs
+
+    case ${machine} in 
```

Comment:
looks like you misunderstood. The setup_target_configs is needed for all machines.
But the content of it is needed by only few. 

Weston is needed only for AM62x. There might be some other ssh configs for example, which is needed by all. So do the machine checks in the function. Not while calling the function.